content/docs/04.5-devx/05-app-profile.md

```diff
@@ -41,8 +41,18 @@ To create your App Profile,
 |Description (optional)   | Description of the App Profile, if any | 
 |Tag (optional)               | Tags on a cluster group are propagated to the cloud/datacenter environments.|
 
-3. **Add Service**: Select a service to start the configuration of the App Profile from the available services and configure the service per the use case.
+3. **Add Service**: Select a service to start configuring the App Profile from the available services and configure the service based on your requirements. The following services are provided:
 
+  * Container Deployment: Containers are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment.
```

Comment:
@resmaa - this description looks good for containers. Will we have a seperate page or section which talks about attributes of the Container deployment?

You can talk to @shubhamrajvanshi incase you need more info about each supported attribute.